housecost.js

```diff
@@ -0,0 +1,25 @@
+let volumeInMeters=8*10*10;
+let gardenSizeInM2=100;
```

Comment:
- it is a good case here to see `let` because you reused the same variables for other values.

- This is fine for now, you have just started learning programming. But I would like to inform you that you should always remember that someone will read your code after you, and it should be clear enough. If someone reads this line he will not understand what is the meaning of these numbers. That's why it is better to declare variables refer to each number like
```js
const wide = 8;
```
or add a comment explaining what is this.
- In programming this called `Magic Numbers` you can google `Magic Numbers` and you will find a lot of articles explaining that.
This is a bit advanced thing to talk about, but good to know from the beginning 👌